testgrid/istio.gen.yaml

```diff
@@ -1,5509 +0,0 @@
-dashboard_groups:
```

Comment:
Is this intentional?

Reply:
No; this change should be in a different PR. This is what I was working on before